examples/docker/cpu/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG PYTHON_VERSION=3.11
+ARG ULTRACK_VERSION=0.6.1
+
+FROM ubuntu:22.04
+
+ARG PYTHON_VERSION
+ARG ULTRACK_VERSION
+
+WORKDIR /app
+
+RUN apt update &&  \
+    apt install -y --no-install-recommends && \
+    apt install -y libgl1-mesa-dev grep curl && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsSL -v -o ~/miniconda.sh -O  "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh" && \
+    chmod +x ~/miniconda.sh && \
+    bash ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    /opt/conda/bin/conda install -y python=${PYTHON_VERSION}
+
+ENV PATH=/opt/conda/bin:$PATH
+
+RUN conda install -y -c conda-forge pyqt uv && \
+    uv pip install --no-cache --system gurobipy higra && \
```

Comment:
@ilan-theodoro do you think we should include napari? I'm in between because using napari with docker is a pain.

Reply:
@JoOkuma i think i had to install napari because we actually use it to read files, right?

Reply:
you're correct